fe/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -981,7 +981,7 @@
     /*
      * Decide how often to check dynamic partition
      */
-    @ConfField(mutable = true, masterOnly = true)
+    @ConfField(mutable = false, masterOnly = true)
```

Comment:
```suggestion
    @ConfField
```

Reply:
I think maybe we should allow dynamic change the check interval seconds?